prismic-model/src/stories-landing.ts

```diff
@@ -10,14 +10,22 @@ const featuredBooks: CustomType = {
   repeatable: false,
   status: true,
   json: {
+    Main: {
+      introText: multiLineText({
+        label: 'Introductory text',
+        placeholder: 'This will appear at the top of the stories landing page.',
+      }),
+    },
     'Featured stories/series': {
-      title,
-      description: multiLineText({ label: 'description' }),
```

Comment:
Have you saved the current contents of these fields? I think they'll get deleted from Prismic when you apply this change?

Reply:
There is no real data at the moment

Reply:
Ah, super. Thanks for checking!